src/rpc.rs

```diff
@@ -134,18 +139,3 @@ impl RpcClient for UPClientZenoh {
         }
     }
 }
-
-#[async_trait]
-impl RpcServer for UPClientZenoh {
```

Comment:
Yeah... I just didn't want to have to update RpcServer! :slightly_smiling_face: 

If you wanna do that first in a separate PR, I think that's fine too :+1:

Reply:
Since the PR might need more discussion and we need to wait for `up-rust`, I just want to ensure your efforts of upgrading `up-rust` can be merged first 🙂
It's fine if you want to focus on the UListener trait first. I can create another PR (maybe next week) to upgrade `up-rust`.